ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -134,65 +143,65 @@ def __init__( self ):
                                Column( 'OpenTickets', Integer, nullable = False, server_default = '0'),
                                Column( 'GocSite', String( 64 ), nullable = False, primary_key = True ),
                                Column( 'Link', String( 1024 ), nullable = False ),
-                               Column( 'LastCheckTime', DateTime, nullable = False ),
+                               Column( 'LastCheckTime', DateTime, nullable = True ),
                                mysql_engine = 'InnoDB' )
 
     JobCache                 = Table( 'JobCache', self.metadata,
                                Column( 'Status', String( 16 ), nullable = False ),
-                               Column( 'Efficiency', DOUBLE, nullable = False, server_default = '0'),
+                               Column( 'Efficiency', FLOAT, nullable = False, server_default = '0'),
```

Comment:
I did not quite understood here, but I do not think you want to use a different type in your column description than the one actually in the DB